scripts/!mods_preload/msu_setup_save_version_system.nut

```diff
@@ -2,41 +2,44 @@ local gt = this.getroottable();
 
 gt.MSU.setupSaveVersionSystem <- function ()
 {
-	gt.MSU.Serialization.Mods <- [];
-	gt.MSU.Serialization.registerMod <- function( _mod, _ver )
-	{
-		if (typeof _mod != "string" || typeof _ver != "string")
+	gt.MSU.Serialization <- {
+		Mods = [],
+
+		function registerMod( _mod, _ver )
```

Comment:
the Debug module already has a registerMod function, perhaps we need to consolidate them